packages/flutter_driver/lib/src/common/fuchsia_compat.dart

```diff
@@ -97,6 +97,6 @@ class FuchsiaCompat {
   static Future<FuchsiaRemoteConnection> connect() async {
     FuchsiaCompat._init();
     return FuchsiaRemoteConnection
-        .connectWithSshCommandRunner(_DummySshCommandRunner());
+        .connectWithSshCommandRunner(_DummySshCommandRunner()); // ignore: invalid_use_of_visible_for_testing_member
```

Comment:
I thought the visibleForTesting/meta etc was removed from FRDP?

Reply:
Ah, maybe it's just that it needs to be rolled into google.  I'll look into that first.